src/wallet/feebumper.cpp

```diff
@@ -80,6 +80,13 @@ static feebumper::Result CheckFeeRate(const CWallet& wallet, const CMutableTrans
         return feebumper::Result::WALLET_ERROR;
     }
 
+    // check that new fee rate does not exceed maxfeerate
```

Comment:
> compare it with the resulting mining score instead of the new transaction’s individual feerate?

🤔 Not sure how to implement the change you suggested honestly.
The current behavour is checking if target fee rate of user wants to bump the transaction to is greater than `maxfeerate` which I believe is the new mining score of the bumped transaction no? 
I might be wrong on my assumption though, would you clarify your comment please? thanks.

Reply:
Yeah, it will probably be easier to introduce that in the context of a _cluster mempool_ where we would much more easily be able to get a read on the effective feerate at which a transaction would be up for block-inclusion.